sql-to-dbsp-compiler/SQL-compiler/src/main/java/org/dbsp/sqlCompiler/compiler/backend/rust/ToRustInnerVisitor.java

```diff
@@ -461,17 +461,20 @@ public VisitDecision preorder(DBSPMapExpression expression) {
             return this.doNullExpression(expression);
         if (expression.getType().mayBeNull)
             this.builder.append("Some(");
-        this.builder.append("BTreeMap::from([")
-                .increase();
+        this.builder.append("Arc::new(BTreeMap::from([");
```

Comment:
Why not `Map::from`, as you do elsewhere?

Reply:
this actually does not work, since the argument is a list of key-value pairs.